packages/manager/apps/exchange/src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>Exchange App</title>
+        <style type="text/css">
+            .ng-cloak {
```

Comment:
consider moving this to css file

Reply:
I'm not sure it is possible. We need to check with @cbourgois but I think it was done that way because if not the styles are loaded too late

Reply:
Yes, if we put this styles in a CSS loaded via webpack it will be loaded too late. It could be in external CSS (as recommended in https://docs.angularjs.org/api/ng/directive/ngCloak) but this CSS should be loaded as soon as possible.
Putting it in the html make it available when the sim is loaded.